tests/python/contrib/test_hexagon/test_meta_schedule.py

```diff
@@ -358,6 +361,7 @@ def test_conv2d_relay_auto_schedule(hexagon_launcher):
         kernel_layout="HWIO",
     )
     mod = tvm.IRModule.from_expr(conv2d + bias)
+    mod = mod.with_attr("executor", relay.backend.Executor("graph"))
```

Comment:
We need `executor = Executor("graph", {"link-params": True})` here.

Reply:
Oh got it! It somehow slipped through